specs/dynamicdependencies/DynamicDependencies.md

```diff
@@ -693,7 +815,9 @@ See [3.2.2. Known Issue: DLL Search Order ignores uap6:AllowExecution](#322-know
 
 ### 5.6.3. Lifecycle Hints are not supported
 
-A package dependency is pinned by a registered package. The lifecycle hints to unpin a package dependency when a file `LifecycleHint_FileOrPath`) or registry key (`LifecycleHint_RegistrySubkey`) is deleted are reserved for future use. Specifying these options today is an error.
+A package dependency is pinned by a registered package. The lifecycle hints to unpin a package
+dependency when a file `LifecycleHint_FileOrPath`) or registry key (`LifecycleHint_RegistrySubkey`)
+is deleted are reserved for future use. Specifying these options today is an error.
```

Comment:
We shouldn't define APIs that don't work yet, they become attractive distractors.  The story for experimental APIs hasn't been fully explored, but once it is, you could put these experimental things in there instead.